### ISSUE TITLE
[python-package] include dmlc_tracker into python-package

### ISSUE THREAD

**Issue by sperlingxx** [dmlc/xgboost#4653]
For now,  `dmlc_tracker/tracker.py` is copied into python-package on the fly, during the python installation. 

```python
# Get dmlc tracker script
shutil.copy('../dmlc-core/tracker/dmlc_tracker/tracker.py', 'xgboost/')
```

Maybe there exists a better approach, include dmlc_tracker into xgboost python-package via `__init__.py`.  Something like: 

```python
try:
    from .dmlc-core.tracker.dmlc_tracker.tracker import RabitTracker
    ....
```

**Comment by sperlingxx** [comment]
@RAMitchell @hcho3  What do you think?  I am grad to work on this.

**Comment by trivialfis** [comment]
@sperlingxx Feel free to open a PR.  :)

**Comment by hcho3** [comment]
Solved by #4731
